### PR TITLE
Implement stripe and Cart menu

### PR DESCRIPTION
**Pull request by maximilian-dz**
Implement Stripe Checkout. All products are added and fetched from stripe
Implement Cart Menu with add/remove functionality. Some styling but not done
Add helper functions to format price, get total item count and cart total value.

### CODE REVIEW

File: src/styles/config/base.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@ html {
 }
 
 body {
-  background-color: #f5f1e8;
+  background-color: rgb(245, 241, 232);
```
Comment (scarsam) [inline src/styles/config/base.scss:6]: @maximilian-dz Variable

File: src/components/cart.js [hunk 2]
```diff
@@ -31,10 +36,26 @@ const Cart = () => {
...
-        {store.count}
+        {items && items.reduce((a, b) => a + (b['quantity'] || 0), 0)}
       </strong>
     </div>
   )
 }
 
 export default Cart
+
+const stripePromise = loadStripe('pk_test_WemDrwL9FdctlL3poeIB3Ilm00b57CDtd8')
```
Comment (scarsam) [inline src/components/cart.js:47]: Är den här nyckeln OK att client?

File: src/components/cart.js [hunk 3]
```diff
@@ -31,10 +36,26 @@ const Cart = () => {
...
+  event.preventDefault()
+  const stripe = await stripePromise
+
+  const { error } = await stripe.redirectToCheckout({
+    items: cart,
+    successUrl: `http://localhost:8000/page-2/`,
+    cancelUrl: `http://localhost:8000/`,
+  })
+  if (error) {
+    console.log('Error:', error)
```
Comment (scarsam) [inline src/components/cart.js:59]: `console.error('Error', error)`

File: src/components/ProductGrid.js [hunk 4]
```diff
@@ -1,22 +1,27 @@
 import React from 'react'
 import Product from './Product'
-import products from '../data/products'
+import { graphql, useStaticQuery } from 'gatsby'
+
+export const ProductGrid = () => {
+  const data = useProducts()
+
+  console.log(data)
```
Comment (scarsam) [inline src/components/ProductGrid.js:8]: Delete

File: src/components/cart.js (hunk 2 above)
Comment (maximilian-dz) [reply src/components/cart.js:130]: Yes!

**Pull request merged**